modules/library/coverage/src/main/java/org/geotools/coverage/grid/io/AbstractGridCoverage2DReader.java

```diff
@@ -117,7 +117,7 @@
     /**
      * Small number used for double comparisons
      */
-    protected static double EPS = 1e-6;
+    protected static double eps = 1e-6;
```

Comment:
Protected methods are public API as well (check if any subclasses are around that can see this before changing case).